Makefile

```diff
@@ -78,5 +78,6 @@ lint:
 .PHONY: test-docker-db test test-ci lint
 
 publish: 
-	docker run -v $(shell pwd):/workspace --workdir /workspace atlas:test kkk ./atlas.toml true
+	@atlas init
```

Comment:
Why do we need the init command here? We haven't merged this yet, so will this work?

Reply:
you need to generate a manifest test it. Will do a followup pr with an example manifest, will use that instead here

Reply:
removed it